### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by thasipablo**
- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Create a directory for your reusable components: /components.
- Install [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- [Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of `<App>`
- Add 2 routes in your application, each should render a page component:
`/`
`/categories`
- Create components inside of the /components directory:
     - Contains individual book state, as well as a button to delete the book
     - Contains a list to renders individual books
     - Contains a form to create new books
     - Contains a navigation to link to the 2 routes you created

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    </div>
+  );
+};
+
+const Categories = () => (
+  <div>
+    <h1>Categories Page</h1>
+  </div>
+);
+
```
Comment (usorfaitheloho) [inline src/App.jsx:33]: Kindly note that there should be two main components in your app, which is the `book` component (containing the components related to a book functionalities), and the `categories` component. so kindly remove this function here and make it a separate component 👍


image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @thasipablo  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Project was initialized with Create React App ✔️
> * Routes were implemented with React Router ✔️

> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @thasipablo  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good job implementing changes
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.jsx (hunk 1 above)
Comment (thasipablo) [reply src/App.jsx]: Hi @usorfaitheloho, I have made a separate component fro Categories.
Thank you.

**Comment by thasipablo** [comment]
Thank you @sumairq

**Pull request merged**